vignettes/baselinenowcast.Rmd

```diff
@@ -216,40 +203,42 @@ target_data <- enw_filter_reference_dates(
   latest_data,
   include_days = n_training_volume - 1
 )
+```
 
-# Get the reporting triangle, adding an additional day because epinowcast
-# we want the max_delay + 1 entries since 0 is a valid delay.
-# This also validates that the data is in the correct format and
-# runs preprocessing see ?enw_preprocess_data for more details
+Next we obtain the reporting triangle, adding an additional day to the
```

Comment:
really odd line breaks here.

Reply:
I think I fixed all the line break issues, they were an artifact of earlier linting that was flagging when my text lines were over 80 characters (it doesn't flag anymore)